internal/internal_utils.go

```diff
@@ -468,10 +468,6 @@ func estimateHistorySize(logger *zap.Logger, event *s.HistoryEvent) int {
 			sum += len(event.SignalExternalWorkflowExecutionInitiatedEventAttributes.Control)
 			sum += len(event.SignalExternalWorkflowExecutionInitiatedEventAttributes.Input)
 		}
-
-	default:
-		logger.Warn("unknown event type", zap.String("Event Type", event.GetEventType().String()))
```

Comment:
Maybe it could be useful to have this log with Debug level?

Reply:
Good idea, let me change that to debug level logs